### DISCUSSION TITLE
UI: translation : OTP validate failed

### DISCUSSION THREAD

**Discussion by caapsoft** [authelia/authelia#6446] [Q&A] [answered]
Congratulations! 

Authelia is the most effective application in this field.

I've notice a lot of effort to make it the best and easiest solution option in the market. We all know how difficult it is, but you are reaching it.

Some aspects needs improvement, and I'd like to suggest one in particular: **translation.** 

For example, this UI message/notification:

**Message**: "The One-Time Password might be wrong"

**File:** /web/src/views/LoginPortal/SecondFactor/OneTimePasswordMethod.tsx **#Line 78**

```
"    onSignInSuccessCallback(res ? res.redirect : undefined);
        } catch (err) {
            console.error(err);
            onSignInErrorCallback(new Error(**"The One-Time Password might be wrong"**));
            setState(State.Failure);
        }
        setPasscode("");"
```

If there is a way to translate, please show me. I tried to find the documentation and suitable entry on template file without success. 
If I'm wrong path, sorry. Please, apply the correct label, change the category or delete this message.``

Thanks!

**Comment by nightah** [comment]
Hey, apologies for the long delay in our response looks like this was buried in my notification history.
That's a great find/pickup, we do provide the means to translate and you can see an example of it below in the same file:

https://github.com/authelia/authelia/blob/master/web/src/views/LoginPortal/SecondFactor/OneTimePasswordMethod.tsx#L49

I'll fix this up so this field is also translated in our next patch release.

**Comment by nightah** [comment, marked as answer]
This will be resolved with the merging of #6950 and the next patch release.

As far as translations go you can sign up to: https://authelia.crowdin.com when the PR is merged to provide translations for your language of choice.
